README.adoc

```diff
@@ -258,7 +392,8 @@ include::finish/system/Dockerfile[]
 
 == Updating {kube} resources
 
-Next, you will update your {kube} deployments to set the environment variables in your containers based on the values configured in the ConfigMap and Secret created previously. 
+Next, you will update your {kube} deployments to set the environment variables in your containers
+based on the values configured in the ConfigMap and Secret created previously. 
```

Comment:
based on the values configured in the ConfigMap and Secret created previously. 
---->
based on the values that are configured in the ConfigMap and Secret that you created previously. 

(not sure if that's the intent but should be more clear if not)